x-pack/plugins/reporting/public/share_context_menu/register_pdf_png_reporting.tsx

```diff
@@ -129,75 +85,57 @@ export const reportingScreenshotShareProvider = ({
     const isV2Job = isJobV2Params(jobProviderOptions);
     const requiresSavedState = !isV2Job;
 
-    const pngReportType = isV2Job ? 'pngV2' : 'png';
+    const pngReportType = 'pngV2';
+    const pdfReportType = isV2Job ? 'printablePdfV2' : 'printablePdf';
 
-    const panelPng = {
-      shareMenuItem: {
-        name: pngPanelTitle,
-        icon: 'document',
-        toolTipContent: licenseToolTipContent,
-        disabled: licenseDisabled || sharingData.reportingDisabled,
-        ['data-test-subj']: 'PNGReports',
-        sortOrder: 10,
-      },
-      panel: {
-        id: 'reportingPngPanel',
-        title: pngPanelTitle,
-        content: (
-          <ScreenCapturePanelContent
+    const openImageModal = () => {
+      const session = overlays.openModal(
+        toMountPoint(
+          <ReportingModalContent
             apiClient={apiClient}
             toasts={toasts}
             uiSettings={uiSettings}
-            reportType={pngReportType}
+            reportType={pngReportType || pdfReportType}
```

Comment:
Is there a different way we could tell if the report is for a PNG or PDF? Both of these variables will be truthy since they are both strings. 

Maybe JobProviderOptions should have `reportType` ?

Reply:
I got rid of report type as a prop since it isnt needed with this refactor - thanks for pointing out the weirdness!